x-pack/plugins/observability_solution/apm/server/routes/transactions/get_transaction_by_trace/index.ts

```diff
@@ -45,11 +80,45 @@ export async function getRootTransactionByTraceId({
           filter: [{ term: { [TRACE_ID]: traceId } }, ...rangeQuery(start, end)],
         },
       },
+      fields: [
```

Comment:
I think this is also only used for redirecting

Reply:
I reduced the field so it should be enough fo the redirect